### PR TITLE
Updated SGD from log (deprecated v1.1) to log_loss (v1.3)

### PR DESCRIPTION
**Pull request by paulleroyza**
These labs have the potential to fail depending on the runtime. the SGD loss has changed from `log` to `log_loss`. deprecated in v1.1 and removed v1.3.  The workbench run already fails. Amendments have been made to all three labs in this set.
----
Log entry from vertex 
/opt/conda/lib/python3.10/site-packages/sklearn/linear_model/_stochastic_gradient.py:163: FutureWarning: The loss 'log' was deprecated in v1.1 and will be removed in version 1.3. Use `loss='log_loss'` which is equivalent.

### CODE REVIEW

**Comment by paulleroyza** [comment]
Update CLA

**Comment by takumiohym** [comment]
Thanks @paulleroyza! 
It seems like there are 2 remaining files to be updated as well. Could you include them? (We have `labs` and `solutions` folders which include the similar code)

https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/master/notebooks/kubeflow_pipelines/pipelines/solutions/trainer_image_vertex/train.py
https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/master/notebooks/kubeflow_pipelines/cicd/solutions/trainer_image_vertex/train.py

https://scikit-learn.org/stable/whats_new/v1.1.html#id7

**Comment by paulleroyza** [comment]
Those files were added, I changed all four, rather than just the original 2 from the previous pull request

**Comment by takumiohym** [comment]
@paulleroyza It seems like files under  `pipelines/labs` and `cicd/labs` are updated, but corresponding files under `pipelines/solutions/` and `cicd/solutions` are not updated. 

There should be 6 files in total.

**Comment by paulleroyza** [comment]
Added those two as well

**Comment by takumiohym** [comment]
@paulleroyza Thanks for adding remaining 2 files! LGTM about the matter.
But it seems like you added other updates after that? I'd appreciate it if you could update them in a separate PR if it is different matter.

**Comment by paulleroyza** [comment]
Reverted those to previous version

**Comment by takumiohym** [comment]
Sounds good. Thanks!
